kubernetes/Earthfile

```diff
@@ -1,15 +1,28 @@
 start:
     LOCALLY
     
-    # Ensure local dependencies are present
-    RUN which kind && which kubectl
+    DO +DEPS
 
     RUN kind create cluster --name earthlydemo
 
-    RUN kubectl apply -f buildkit.yaml
-    RUN kubectl wait --for=condition=ready pod -l component=buildkitd -l app=earthly --timeout=90s
+    RUN kubectl apply -f ./buildkit.yaml
+    RUN kubectl wait --for=condition=available deployment/earthly-buildkitd              --timeout 90s
+    RUN kubectl wait --for=condition=ready     pod -l component=buildkitd -l app=earthly --timeout=90s
```

Comment:
I understand that the formatting here makes it line up with the timeout flag from above, if that's intentional maybe the extra spaces should be done immediately before?
```suggestion
    RUN kubectl wait --for=condition=ready pod -l component=buildkitd -l app=earthly     --timeout=90s
```

Reply:
Not just to make the timeouts align. Also to align the condition/selector part into one column too. This is the more extreme version:

```
RUN kubectl wait --for=condition=available deployment/earthly-buildkitd                                        --timeout 90s
 RUN kubectl wait --for=condition=ready     pod                          -l component=buildkitd -l app=earthly  --timeout=90s
```

Reply:
Here's what the alignment looks like on my terminal:

![image](https://user-images.githubusercontent.com/1806823/126012005-8740c9e7-f73f-4a3a-8003-68d7cf626a90.png)

The third wait command doesn't appear to line up.

Reply:
The third wait was on its own line, lining that up was weird to me.

Ill ditch all attempts at alignment.